VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx

```diff
@@ -52,56 +35,16 @@ const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ c
       return getFilteredNoMatch()
     }
 
-    switch (currentTab) {
-      case PrescriptionHistoryTabConstants.ALL:
-        return (
-          <>
-            <TextView textAlign={'center'} variant="MobileBodyBold">
-              {t('prescription.history.empty.title')}
-            </TextView>
-            <TextView
-              pt={theme.dimensions.condensedMarginBetween}
-              textAlign={'center'}
-              variant="MobileBody"
-              accessibilityLabel={a11yLabelVA(t('prescription.history.empty.message'))}>
-              {t('prescription.history.empty.message')}
-            </TextView>
-          </>
-        )
-      case PrescriptionHistoryTabConstants.PENDING:
-        return (
-          <>
-            <TextView textAlign={'center'} variant="MobileBodyBold">
-              {t('prescription.history.empty.pending.title')}
-            </TextView>
-            <TextView
-              pt={theme.dimensions.condensedMarginBetween}
-              textAlign={'center'}
-              variant="MobileBody"
-              accessibilityLabel={a11yLabelVA(t('prescription.history.empty.pending.message'))}>
-              {t('prescription.history.empty.pending.message')}
-            </TextView>
-          </>
-        )
-      case PrescriptionHistoryTabConstants.TRACKING:
-        return (
-          <>
-            <TextView textAlign={'center'} variant="MobileBodyBold">
-              {t('prescription.history.empty.tracking.title')}
-            </TextView>
-            <TextView pt={theme.dimensions.condensedMarginBetween} textAlign={'center'} variant="MobileBody">
-              {t('prescription.history.empty.tracking.p1')}
-            </TextView>
-            <TextView
-              pt={theme.dimensions.condensedMarginBetween}
-              textAlign={'center'}
-              variant="MobileBody"
-              accessibilityLabel={a11yLabelVA(t('prescription.history.empty.tracking.p2'))}>
-              {t('prescription.history.empty.tracking.p2')}
-            </TextView>
-          </>
-        )
-    }
+    return (
+      <>
+        <TextView textAlign={'center'} variant="MobileBodyBold">
```

Comment:
shouldn't this also be a header, I know it wasn't previously

Reply:
Make sense to me, added the accessibilityRole